app/models/manageiq/providers/vmware/cloud_manager/vm/operations/power.rb

```diff
@@ -0,0 +1,25 @@
+module ManageIQ::Providers::Vmware::CloudManager::Vm::Operations::Power
+  def validate_pause
+    validate_unsupported("Pause operation")
+  end
+
+  def raw_start
+    with_provider_object(&:power_on)
+    self.update_attributes!(:raw_power_state => "on")
```

Comment:
@agrare Is this how you suggested to update the power state? One thing I noticed while testing in the UI was that the change is not completely instantaneous - it seems like it is propagated when the refresh parser does it's job. I am not sure if this is the planned behaviour or is there something wrong.

It does, however, have effect. If I don't update the power state attr, then I have to manually request a refresh after the power state changes in the vcloud itself.